data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -317,24 +337,17 @@ then
 	touch cco-bootstrap.done
 fi
 
-# Wait for the etcd cluster to come up.
-until bootkube_podman_run \
-		--rm \
-		--name etcdctl \
-		--env ETCDCTL_API=3 \
-		--volume /opt/openshift/tls:/opt/openshift/tls:ro,z \
-		--entrypoint etcdctl \
-		"${MACHINE_CONFIG_ETCD_IMAGE}" \
-		--dial-timeout=10m \
-		--cacert=/opt/openshift/tls/etcd-ca-bundle.crt \
-		--cert=/opt/openshift/tls/etcd-client.crt \
-		--key=/opt/openshift/tls/etcd-client.key \
-		--endpoints="${ETCD_ENDPOINTS}" \
-		endpoint health
-do
-	echo "etcdctl failed. Retrying in 5 seconds..."
-	sleep 5
-done
+# in case of single node, if we removed etcd, there is no point to wait for it on restart
+{{- if .BootstrapInPlace}}
+if [ ! -f stop-etcd.done ]
```

Comment:
Is there any reason this test should only be run when bootstrapping in place?

Reply:
Yes, because the bootstrap-in-place.sh takes etcd down once cluster-bootstrap is done.
So in case bootstrap-in-place.sh failed or timed out bootkube unit gets restarted.